src/test/java/org/openmainframeproject/cobolcheck/MockIT.java

```diff
@@ -825,6 +856,8 @@ public void it_inserts_call_mocks_without_commas_correctly() throws IOException
             "                WHEN \"Mocking tests\"                                            " + Constants.NEWLINE +
```

Comment:
There is missing a perform process-unmock-call after the commented call to bogus above these lines.